src/averageAges.js

```diff
@@ -20,6 +20,28 @@ function calculateMenAverageAge(people, century) {
   // avoid using loop and forEach
   // replace `if ()` statement with &&, || or ?:
   // without nesting
+  people.forEach(function(person) {
+    person.age = person.died - person.born;
```

Comment:
Avoid using forEach
You can use `person.died - person.born` in your reduce method